services-js/access-boston/src/server/access-boston.ts

```diff
@@ -90,8 +90,32 @@ export async function makeServer(port) {
   await server.register(acceptLanguagePlugin);
   await server.register(Crumb);
 
-  // Required by both auth schemes
-  await server.register(cookieAuthPlugin);
+  if (
+    process.env.NODE_ENV === 'production' &&
+    !process.env.SESSION_COOKIE_PASSWORD
+  ) {
+    throw new Error('Must set $SESSION_COOKIE_PASSWORD in production');
+  }
+
+  // TODO(finh): Add Redis support for session storage
+  await server.register({
+    plugin: yar,
+    options: {
+      // Always stores everything in the cache, so we can clear out sessions
+      // unilaterally rather than relying on cookie expiration and being
+      // vulnerable to replays.
+      maxCookieSize: 0,
+      cookieOptions: {
+        password:
+          process.env.SESSION_COOKIE_PASSWORD ||
+          'iWIMwE69HJj9GQcHfCiu2TVyZoVxvYoU',
```

Comment:
Do we want this dummy string to go into production?

Reply:
Yeah, it's fine, so we can run w/o any configuration. I can change it to be like "test-xxxx" or "fake-xxxx" so it's obvious that it's on purpose?

There's code that enforces that in prod the password is set in the environment.